matchms/filtering/metadata_processing/repair_not_matching_annotation.py

```diff
@@ -42,29 +43,32 @@ def repair_not_matching_annotation(spectrum_in: Spectrum):
 
     spectrum = spectrum_in.clone()
 
+    smiles = spectrum.get("smiles")
+    inchi = spectrum.get("inchi")
+    inchikey = spectrum.get("inchikey")
+
     if not _check_fully_annotated(spectrum):
-        if not is_valid_inchi(spectrum.get("inchi")) and \
-           not is_valid_smiles(spectrum.get("smiles")) and \
-           not is_valid_smiles(spectrum.get("inchikey")):
```

Comment:
Just realized this now --> that should be `is_valid_inchikey` !

Reply:
Ah good catch! This probably happened when I manually resolved the merge conflicts